.github/CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 # .github directory & its subdirectories
-/.github/ @newrelic/developer-enablement
+/.github/ @newrelic/docs-engineering
 
 # all javascript files
-*.js @newrelic/developer-enablement
+*.js @newrelic/docs-engineering
 
-yarn.lock @newrelic/developer-enablement
+yarn.lock @newrelic/docs-engineering
 
 # developer site terms, notify interested parties of changes
-/src/markdown-pages/terms.mdx @newrelic/developer-enablement @abrunner94 @caylahamann
+/src/markdown-pages/terms.mdx @newrelic/docs-engineering @abrunner94 @caylahamann
```

Comment:
slight optimization you can make. i think cayla will be on docs-engineering so you can remove the user.

🙈

Reply:
Fantastic 👍🏻